docs/IE_DG/Int8Inference.md

```diff
@@ -111,17 +63,27 @@ available from the Inference Engine API. The layers have the following marks:
-inception_5b/output_oScale... EXECUTED       layerType: ScaleShift         realTime: 1390       cpu: 1390           execType: jit_avx2_FP32
-inception_5b/output_oScale... EXECUTED       layerType: Reorder            realTime: 143        cpu: 143            execType: reorder_FP32
-inception_5b/pool             EXECUTED       layerType: Pooling            realTime: 59301      cpu: 59301          execType: ref_any_I8
-```
 
-The `execType` column of the table includes inference primitives with specific suffixes.
+| layerName                                                 | execStatus | layerType    | execType             | realTime (ms) | cpuTime (ms) |
+| --------------------------------------------------------- | ---------- | ------------ | -------------------- | ------------- | ------------ |
+| resnet\_model/batch\_normalization\_15/FusedBatchNorm/Add | EXECUTED   | Convolution  | jit\_avx512\_1x1\_I8 | 0.377         | 0.377        |
+| resnet\_model/conv2d\_16/Conv2D/fq\_input\_0              | NOT\_RUN   | FakeQuantize | undef                | 0             | 0            |
+| resnet\_model/batch\_normalization\_16/FusedBatchNorm/Add | EXECUTED   | Convolution  | jit\_avx512\_I8      | 0.499         | 0.499        |
+| resnet\_model/conv2d\_17/Conv2D/fq\_input\_0              | NOT\_RUN   | FakeQuantize | undef                | 0             | 0            |
+| resnet\_model/batch\_normalization\_17/FusedBatchNorm/Add | EXECUTED   | Convolution  | jit\_avx512\_1x1\_I8 | 0.399         | 0.399        |
+| resnet\_model/add\_4/fq\_input\_0                         | NOT\_RUN   | FakeQuantize | undef                | 0             | 0            |
+| resnet\_model/add\_4                                      | NOT\_RUN   | Eltwise      | undef                | 0             | 0            |
+| resnet\_model/add\_5/fq\_input\_1                         | NOT\_RUN   | FakeQuantize | undef                | 0             | 0            |
+
+
+> The `exeStatus` column of the table includes possible values:
+> - `EXECUTED` - layer was executed by standalone primitive,
+> - `NOT_RUN` - layer was not executed by standalone primitive or was fused with another operation and executed in another layer primitive.  
+>
+> The `execType` column of the table includes inference primitives with specific suffixes.  
+
+`FakeQuantize` layers were fused with previous `Convolution` layers and executed in one `Convolution` primitive. `Convolution` layers were executed with input integer tensors with using integer vectorized instructions. In the same primitive, `Convolution` operation result is used as input for `FakeQuantize` operation, which is fused in `Convolution` layer primitive. `FakeQuantize` operation was executed by float pointing CPU instructions in one CPU plugin `Convolution` primitive. As result inference was made with maximum performance.
```

Comment:
We have to simplify the statement: All Convolution layers are executed in int8 precision. Rest layers are fused into Convolutions using post ops optimization technic: https://github.com/openvinotoolkit/openvino/blob/19953ce1ff854b9c993a60972391b5aef015e7c8/docs/IE_DG/supported_plugins/CPU.md#internal-cpu-plugin-optimizations

Reply:
thanks, done!